compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -702,6 +702,12 @@ static LogicalResult setMatmulPadRootConfig(
                                          workgroupTileSizes.end());
   parallelTileSizes.back() = 0;
 
+  // Clamp inner tiling sizes to avoid masking.
```

Comment:
Probably good to mentioned the why (keeping lowering_config consistent with the actual padding/tiling applied)?